### ISSUE TITLE
Cannot read property '__VUE_I18N__' of null at updateComponentTreeTags

### ISSUE THREAD

**Issue by aj-dev** [intlify/vue-i18n-next#601]
### Reporting a bug?

This error is being thrown from `vue-devtools-plugin-vue-i18n`. For some reason `instance.vnode.el` can be null when vnode is RouterView. See attached screenshot.

A quick fix would be to do this `if (instance && instance.vnode.el?.__VUE_I18N__)` or `if (instance && instance.vnode.el && instance.vnode.el.__VUE_I18N__)`.

### Expected behavior

No errors in dev tools console

### Reproduction

Create  nested routes where parent route component is SFC with only `<router-view />` in template block.

### System Info

```shell
System:
    OS: macOS 11.4
    CPU: (8) x64 Intel(R) Core(TM) i7-4770HQ CPU @ 2.20GHz
    Memory: 111.26 MB / 16.00 GB
    Shell: 3.2.57 - /bin/bash
  Binaries:
    Node: 14.17.0 - ~/.volta/tools/image/node/14.17.0/bin/node
    npm: 6.14.13 - ~/.volta/tools/image/node/14.17.0/bin/npm
  Browsers:
    Brave Browser: 92.1.27.108
    Chrome: 92.0.4515.107
    Firefox: 88.0
    Safari: 14.1.1
  npmPackages:
    vue: ^3.1.5 => 3.1.5 
    vue-i18n: ^9.2.0-beta.1 => 9.2.0-beta.1
```

### Screenshot

<img width="738" alt="Screenshot 2021-07-29 at 09 01 51" src="https://user-images.githubusercontent.com/2584733/127448365-6c279de9-38cd-4aae-811b-fee0e0e68565.png">
<img width="714" alt="Screenshot 2021-07-29 at 09 03 30" src="https://user-images.githubusercontent.com/2584733/127448376-4f04e315-f543-4406-b758-b57bda2d652c.png">

### Validations

- [X] Read the [Contributing Guidelines](https://github.com/intlify/vue-i18n-next/blob/master/.github/CONTRIBUTING.md)
- [X] Read the [Documentation](https://vue-i18n.intlify.dev/)
- [X] Check that there isn't already an issue that reports the same bug to avoid creating a duplicate.
- [X] Check that this is a concrete bug. For Q&A open a [GitHub Discussion](https://github.com/intlify/vue-i18n-next/discussions)

**Comment by kazupon** [comment]
Thank you for your reporting!
I've just fixed it!
Please check it when we release v9.2-beta.2.

**Comment by aj-dev** [comment]
@kazupon thanks for the fix! I can confirm that the issue is solved in v9.2-beta.2.

**Comment by kazupon** [comment]
@aj-dev 
I'm glad to hear about it 😄 
Happy coding!

**Comment by mitow7821** [comment]
I am getting this error while using t() with keep alive : 
![image](https://user-images.githubusercontent.com/34492112/129579186-addf4857-02f5-4306-9711-d678495b8e9d.png)

**Comment by mreduar** [comment]
@kazupon When will this be added to the stable version?

**Comment by kazupon** [comment]
I'm working on nuxt/i18n with the nuxt community.
when nuxt/i18n v8 beta is released, I'll release the vue-i18n v9.2 rc version
